frontend/src/pages/Onboarding.tsx

```diff
@@ -0,0 +1,172 @@
+import { useState, ChangeEvent } from "react";
+import { Button, ButtonGroup, Flex, Spacer, Input, Heading, VStack, InputGroup, Textarea, FormLabel } from '@chakra-ui/react';
+
+const Onboarding = () => {
+    let [userInfo, setUserInfo] = useState({
+        role: '',
+        bio:'',
+        expertise: '',
+        learningGoals: '',
+        experience: '',
+        jobTitle: '',
+        interests: '',
+        hourlyRate: ''
+    });
```

Comment:
Create an interface for this data

Reply:
Ended up create additional states for Mentor and Mentee and changing the existing state to track which role was chosen in the initial question. Created an interface for each.